### ISSUE TITLE
"disableVisualSelection: current" is ignored work

### ISSUE THREAD

**Issue by RichTeaTime** [handsontable/handsontable-pro#222]
### Description
Docs describe "disableVisualSelection" possible values:
> • true - Disables any type of visual selection (current and area selection),
> •	false - Enables any type of visual selection. This is default value.
> •	'current' - Disables the selection of a currently selected cell, the area selection is still present.
> •	'area' - Disables the area selection, the currently selected cell selection is still present.
> •	'header' - Disables the headers selection, the currently selected cell selection is still present.

However, calls to ```this.highlight.isEnabledFor(_highlight.CELL_TYPE)``` fail to match 'current' which is clearly what's intended.

### Steps to reproduce
1. Set disableVisualSelection: 'current' or ['current'...]
2. Run and select a cell
EXPECTED: The current cell is not highlighted, as requested.
ACTUAL: The current cell is still highlighted, the disableVisualSelection value is ignored.

3. Set disableVisualSelection: 'cell' or ['cell']
4. Run and select a cell
EXPECTED: The current cell is highlighted, the disableVisualSelection value is ignored.
ACTUAL: The current cell is not highlighted.

### Demo
https://jsfiddle.net/v5wxdfzq/

### Your environment
* Handsontable version: 6.2.2
* Browser Name and version: Chrome 72.0.3626.109
* Operating System: Windows 10

**Comment by RichTeaTime** [comment]
Not sure how to submit a pull request on this, but changing ```highlight.isEnabledFor``` from:

      var type = highlightType === 'current' ? CELL_TYPE : highlightType;

to

      var type = highlightType === CELL_TYPE ? 'current' : highlightType;

would probably do it.

**Comment by AMBudnik** [comment]
You're right. It does not work. Half of the options got broken after introducing v `2.0.0` Thank you for sharing @RichTeaTime
